napari/layers/_shapes_layer/model.py

```diff
@@ -1183,6 +1184,46 @@ def _move(self, coord):
                 self._drag_box = np.array([self._drag_start, coord])
                 self._set_highlight()
 
+    def to_svg(self, canvas_shape=None, shape_type=None):
+        """Returns an svg string with all the shapes contained in the layer.
+        Passing a `shape_type` argument leads to only shapes from that
+        particular `shape_type` being returned.
+
+        Parameters
+        ----------
+        canvas_shape : np.ndarray | tuple | None
+            2-tuple defining shape of svg canvas to be generated. If non
```

Comment:
```suggestion
            2-tuple defining shape of svg canvas to be generated. If not
```
It's also standard to include the length of the tuple up top, and to append `, optional` in place of `| None`. Furthermore, typing in documentation is typically "duck-typed"/non-strict so you could just list it as a `2-tuple` and not include `np.ndarray` or specify it as a `2-sequence`

Reply:
fixed